### ISSUE TITLE
Distinguish network with labels when watch more than one networks gradients and weights

### ISSUE THREAD

**Issue by Fitz13009** [wandb/client#2284]
Hi, 

I am asking for this new feature because I got quite confused when I watch multiple networks at the same time in one run. In my case there are 4 different networks (nn.Modules) and each of them have multiple Linear Layers. Thus when I look at e.g. the gradients in the dashboard, there will be 30+ figures without meaningful caption on it. 

**Describe the solution you'd like**
Add one more function argument "captions" to wandb.watch, so that it will assign the captions to networks respectively.

Thanks a lot!

Best,
Ge Li

**Comment by vanpelt** [comment]
Thanks for the feature request, we'll track it here.
